log/log.go

```diff
@@ -23,6 +23,7 @@ type Logger interface {
 	Fatalw(msg string, keyvals ...interface{})
 	Panicw(msg string, keyvals ...interface{})
 	With(args ...interface{}) *zap.SugaredLogger
+	Named(s string) *zap.SugaredLogger
```

Comment:
Any chance to return the interface instead of the concrete type here ?

Reply:
I inspired myself of the `With` function, but I guess I can try and change them both. I'll take a look on Monday.